.ci/deps.sh

```diff
@@ -34,7 +34,7 @@ npm install
 mkdir -p ~/.RLibrary
 echo '.libPaths( c( "~/.RLibrary", .libPaths()) )' >> .Rprofile
 echo 'options(repos=structure(c(CRAN="http://cran.rstudio.com")))' >> .Rprofile
-R -e "install.packages('lintr', dependencies=TRUE, quiet=TRUE, verbose=FALSE)"
+R -e 'install.packages("formatR", dependencies=TRUE, quiet=TRUE, verbose=FALSE)'
```

Comment:
btw is this commit in the history behind the other commits in this PR? as the others depend on this one, move it please if needed :+1: